src/doom/m_menu.c

```diff
@@ -1456,7 +1458,7 @@ static void M_DrawCrispnessHeader(const char *item)
 {
     M_snprintf(crispy_menu_text, sizeof(crispy_menu_text),
                "%s%s", crstr[CR_GOLD], item);
-    M_WriteText(ORIGWIDTH/2 - M_StringWidth(item) / 2, 12, crispy_menu_text);
+    M_WriteText(ORIGWIDTH/2 - M_StringWidth(item) / 2, 2, crispy_menu_text);
```

Comment:
A line is 8px high, right? This would leave us 4px here.

Reply:
Talking about this?
https://github.com/fabiangreffrath/crispy-doom/blob/d2fafb59da03a83c86fc732b82d00a408b4c4874/src/doom/m_menu.c#L130

Regardless, the argument could certainly be made that a value of 4px looks a little better:
<img src="https://user-images.githubusercontent.com/43701387/210284641-0bc45fe8-aad9-413b-b36b-f00d31cd1342.png" width=50% height=50%>

Reply:
I think 6px looks the best:
<img src="https://user-images.githubusercontent.com/43701387/210474821-045243ba-4f8b-41e3-93e1-d29560c58b3e.png" width=50% height=50%>
Any opinion on the matter?

Reply:
To throw a wrench in things, is the "uncapped framerate" option redundant now?

Seems like it's covered by:
- fps limit: 35
- fps limit: none, 36-500

Reply:
> I think 6px looks the best:
> ...
> Any opinion on the matter?

Yes, that's fine, thanks!